src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+// En src/redux/categories/categoriesSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {
+    checkStatus: (state) => state,
+  },
+});
```

Comment:
- [ ] As per the last code review, kindly make sure that the initialState is set to an empty array.
- [ ] Please make sure to check the status first before returning the string. As the following:
```javascript
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```